<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterface</class>
 <widget class="QMainWindow" name="UserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>845</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Option price simulator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(159, 160, 193);
background-color: rgb(189, 185, 238)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1231</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="inputparameterstext_2" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <widget class="QWidget" name="inputparameterstext_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(13, 13, 20);</string>
      </property>
      <property name="text">
       <string>Simulation results</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="priceShowWidget" native="true">
    <property name="geometry">
     <rect>
      <x>691</x>
      <y>394</y>
      <width>284</width>
      <height>212</height>
     </rect>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>271</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>European put price:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>placeholder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>European call price:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>plcerholder</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>Binary put price:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>placeholder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>Binary call price:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>placeholder</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>Average exercise price:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>placeholder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>450</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
background-color:rgb(142, 137, 222)</string>
    </property>
    <property name="text">
     <string>Run Monte-Carlo
simulation</string>
    </property>
   </widget>
   <widget class="QWidget" name="priceShowWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1371</x>
      <y>1004</y>
      <width>284</width>
      <height>212</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>95</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>European put price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>9</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>placeholder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>49</y>
       <width>94</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>European call price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>49</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>plcerholder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>89</y>
       <width>79</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>Binary put price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>89</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>placeholder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>129</y>
       <width>78</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>Binary call price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>129</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>placeholder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>169</y>
       <width>114</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>Average exercise price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>169</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>placeholder</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>610</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(13, 13, 20);</string>
    </property>
    <property name="text">
     <string>Black-Scholes formula results</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>680</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
background-color:rgb(142, 137, 222)</string>
    </property>
    <property name="text">
     <string>Calculate with Black-Scholes
formula</string>
    </property>
   </widget>
   <widget class="QWidget" name="priceShowWidget_3" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>640</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_31">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>European put price:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_32">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>placeholder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_33">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>European call price:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_34">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>plcerholder</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_35">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>Binary put price:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_36">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>placeholder</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_37">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>Binary call price:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_38">
       <property name="styleSheet">
        <string notr="true">color:rgb(13, 13, 20);</string>
       </property>
       <property name="text">
        <string>placeholder</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="inputparameterstext_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>3</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(13, 13, 20);</string>
     </property>
     <property name="text">
      <string>Price trajectories</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>415</y>
      <width>471</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="inputparameterstext" native="true">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>0</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(13, 13, 20);</string>
        </property>
        <property name="text">
         <string>Input Parameters</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="inputWidget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_21">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>Timestep (days):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>131</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="timeStep">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_22">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>Maturity (years):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="4">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>131</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="maturity">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QLabel" name="label_23">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>Instrument strike price
at the beginning (unit of currency):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>70</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QLabel" name="label_25">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>Option strike price at the beginning (unit of currency):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>70</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="5">
         <widget class="QLineEdit" name="K">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>90.0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_24">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>Volatility (%):</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="4">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>131</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="5">
         <widget class="QLineEdit" name="volatility">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QLabel" name="label_27">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>Risk-free interest rate (%):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="2">
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>90</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="5">
         <widget class="QLineEdit" name="riskFree">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>3.3</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" rowspan="2" colspan="4">
         <widget class="QLabel" name="label_28">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>Number of runs:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="2" colspan="3">
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>111</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="5">
         <widget class="QLineEdit" name="NOofRuns">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLineEdit" name="S_0">
          <property name="styleSheet">
           <string notr="true">color:rgb(13, 13, 20);</string>
          </property>
          <property name="text">
           <string>100.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>inputWidget</zorder>
   <zorder>priceShowWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>inputparameterstext</zorder>
   <zorder>plotWidget</zorder>
   <zorder>inputparameterstext_2</zorder>
   <zorder>priceShowWidget_2</zorder>
   <zorder>label_30</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>priceShowWidget_3</zorder>
   <zorder>inputparameterstext_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
